cypress/e2e/experiments.cy.ts

```diff
@@ -65,8 +65,8 @@ describe('Experiments', () => {
             .type(secondaryMetricName)
             .should('have.value', secondaryMetricName)
         cy.get('[data-attr="metrics-selector"]').click()
-        cy.contains('Trends').should('be.visible')
-        cy.contains('Funnels').should('be.visible')
+        cy.contains('Funnels').should('exist')
+        cy.contains('Trends').should('exist')
```

Comment:
Cypress for some reason no longer sees them. Changing them to 'exist' for now, I've confirmed they're still visible.